tests/promo-code/promo-code.query.test.js

```diff
@@ -8,4 +13,11 @@ describe('Promo-codes queries', () => {
     expect(answers).toBeDefined();
     expect(answers.items).toBeInstanceOf(Array);
   });
+
+  it('Should get promo-code by field code', async () => {
+    const operations = await setupApp();
+    await addPromoCode(newPromoCode, operations);
+    const promoCode = await getPromoCodeByCode(newPromoCode.code, operations);
```

Comment:
empty line

Reply:
please, test also case when promocode is not found

Reply:
Ok, got it

Reply:
Done